assets/src/dashboard/app/views/editorSettings/publisherLogo/index.js

```diff
@@ -42,7 +47,9 @@ const TEXT = {
   ),
   SUBMIT: __('Upload', 'web-stories'),
 };
-function PublisherLogoSettings() {
+
+// eslint-disable-next-line no-unused-vars
+function PublisherLogoSettings({ onUpdatePublisherLogo, publisherLogos }) {
```

Comment:
will use these once OK to connect to fileUpload component. Wanted to pass them through to set up propTypes here.